layers/meta-balena-jetson/contracts/sw.package/yocto/ipk/tegra-libraries/contract.json

```diff
@@ -0,0 +1,25 @@
+{
+  "slug": "tegra-libraries",
+  "type": "sw.package.yocto.ipk",
+  "name": "Package list that provide Tegra GPU support",
+  "version": "1.0.0",
+  "composedOf": {
```

Comment:
The idea is that `composedOf` is an array of contracts, but maybe we want to have some way to just reference other contracts? Maybe using a partial contract that is sufficient to identify what we are referring to (eg. slug + type + version)